src/test/java/org/springframework/data/elasticsearch/core/geo/ElasticsearchTemplateGeoTests.java

```diff
@@ -172,7 +173,7 @@ public void shouldFindStringAnnotatedGeoMarkersInRangeForGivenCriteriaQuery() {
 		CriteriaQuery geoLocationCriteriaQuery = new CriteriaQuery(
 				new Criteria("locationAsString").within(new GeoPoint(51.000000, 0.100000), "1km"));
 		// when
-		List<LocationMarkerEntity> geoAuthorsForGeoCriteria = operations.queryForList(geoLocationCriteriaQuery,
+		List<SearchHit<LocationMarkerEntity>> geoAuthorsForGeoCriteria = operations.queryForList(geoLocationCriteriaQuery,
```

Comment:
Once we have `SearchHits`, the migration will be likely easier when users can switch from `List<T>` to `SearchHits<T>`.